helm/charts/kratos/templates/deployment.yaml

```diff
@@ -42,7 +42,7 @@ spec:
     {{- if .Values.deployment.extraInitContainers}}
       initContainers:
         {{- if .Values.deployment.extraInitContainers }}
-{{ toYaml ( tpl .Values.deployment.extraInitContainers . ) | indent 8 }}
+{{ tpl .Values.deployment.extraInitContainers . | indent 8 }}
```

Comment:
I think we should leave this the way it was before https://github.com/ory/k8s/pull/300 to avoid changing behavior too much. Again, I don't like using `tpl` because that makes the values.yaml file look weird, but I prefer consistency.

Reply:
The idea of using `tpl` is a common in some charts, that's why we use it. For example [here](https://github.com/helm/charts/blob/master/incubator/elasticsearch/ci/extrainitcontainers-values.yaml)